README.rst

```diff
@@ -4,28 +4,85 @@ eventyay-tickets
 .. image:: https://codecov.io/gh/fossasia/eventyay-tickets/branch/master/graph/badge.svg
    :target: https://codecov.io/gh/pretix/pretix
 
-
 Project status & release cycle
 ------------------------------
 
-Welcome to the eventyay project. The ticketing component of the system provides options for ticket sales and event related items such a Tshirts. Eventyay is developed since 2014. The ticketing component is based on a fork of pretix.
+Welcome to the **Eventyay** project! The ticketing component of the system provides options for **ticket sales and event-related items** such as T-shirts. Eventyay has been in development since **2014** and is based on a fork of **Pretix**.
 
 Support
 -------
 
-This project is free and open source software. Professional support is available to customers of the hosted eventyay service or eventyay enterprise offerings. If you are interested in commercial support, hosting services or supporting this project financially, please go to `eventyay.com`.
+This project is **free and open-source software**. Professional support is available to customers of the **hosted Eventyay service** or **Eventyay enterprise offerings**. If you are interested in commercial support, hosting services, or supporting this project financially, please go to `eventyay.com`.
 
 Contributing
 ------------
+
 Please look through our issues and start contributing.
 
+### Setting Up Eventyay-Tickets Locally
+Since **Eventyay-Tickets** is a backend-driven project, you need **Docker** to run it properly. If you already have Docker installed, skip to the next step.
+
+1. **Install Docker (If Not Installed)**
+   - [Download Docker Desktop](https://www.docker.com/products/docker-desktop/) and install it on your system.
+   - Ensure Docker is running by checking the version:
+     ```sh
+     docker --version
+     ```
+
+2. **Clone & Set Up Eventyay-Docker**
+   Eventyay-Tickets runs inside **Eventyay-Docker**, so you need to set that up as well:
+   ```sh
+   cd ..  # Go back to the parent directory
+   git clone https://github.com/fossasia/eventyay-docker.git
+   cd eventyay-docker
+   ```
+
+3. **Modify Hosts File (Important!)**
```

Comment:
Don't consider it "important". The development environment should work with `localhost` domain. It will be fixed in the future.